plugins/woocommerce/src/Internal/Admin/SystemStatusReport.php

```diff
@@ -0,0 +1,215 @@
+<?php
+/**
+ * Add additional system status report sections.
+ */
+
+namespace Automattic\WooCommerce\Internal\Admin;
+
+use Automattic\WooCommerce\Admin\Notes\Notes;
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * SystemStatusReport class.
+ */
+class SystemStatusReport {
+	/**
+	 * Class instance.
+	 *
+	 * @var SystemStatus instance
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 */
+	public static function get_instance() {
+		if ( ! self::$instance ) {
+			self::$instance = new self();
+		}
+		return self::$instance;
+	}
+
+	/**
+	 * Hook into WooCommerce.
+	 */
+	public function __construct() {
+		add_action( 'woocommerce_system_status_report', array( $this, 'system_status_report' ) );
+
```

Comment:
Nitpick: extra space
```suggestion
```

Reply:
Removed in 902dcf3369212732fcb9490260660b459e888f08